resources/globalconfig.rb

```diff
@@ -77,7 +77,7 @@
 property :installonly_limit, String, regex: [/^\d+/, /keep/], default: '3', description: "Number of packages listed in installonlypkgs to keep installed at the same time. Setting to 0 disables this feature. Default is '0'. Note that this functionality used to be in the 'installonlyn' plugin, where this option was altered via tokeep.  Note that as of version 3.2.24, yum will now look in the yumdb for a installonly attribute on installed packages. If that attribute is 'keep', then they will never be removed."
 property :installonlypkgs, String, description: 'List of package provides that should only ever be installed, never updated.  Kernels in particular fall into this category. Defaults to kernel, kernel-bigmem, kernel-enterprise, kernel-smp, kernel-modules, kernel-debug, kernel- unsupported, kernel-source, kernel-devel, kernel-PAE, kernel- PAE-debug.'
 property :installroot, String, description: 'Specifies an alternative installroot, relative to which all packages will be installed.'
-property :ip_resolve, Integer, equal_to: %w(4 6), description: "Determines how yum resolves host names. '4': resolve to IPv4 addresses only. '6': resolve to IPv6 addresses only."
+property :ip_resolve, String, equal_to: %w(4 6), description: "Determines how yum resolves host names. '4': resolve to IPv4 addresses only. '6': resolve to IPv6 addresses only."
```

Comment:
Why are you changing this to a `String`? It probably should support either that or `Integer`.

Reply:
The options are used as a string, it's a version string rather than an integer.